ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Protocol/PBFT.hs

```diff
@@ -438,7 +440,7 @@ append PBftConfig{} PBftWindowParams{..} =
 -- avoid space leaks.
 data PBftValidationErr c
   = PBftInvalidSignature !Text
-  | PBftNotGenesisDelegate !(PBftVerKeyHash c) !(PBftLedgerView c)
+  | PBftNotGenesisDelegate !(PBftVerKeyHash c) !(Ticked (PBftLedgerView c))
```

Comment:
This is the only aspect of the diff that I anticipate trickling down into eg `cardano-node`.